scipy/optimize/_differentialevolution.py

```diff
@@ -291,6 +373,14 @@ class DifferentialEvolutionSolver(object):
 
             - 'latinhypercube'
             - 'random'
+    workers : pool object or int, optional
+        Optional iterable object which is used for parallelization. It can be
+        any object with a map method that follows the same calling sequence as
+        the built-in map function, and with poolsize method.
```

Comment:
It's not clear what the poolsize method is. What does it return? What are it's arguments?.  Perhaps we need to give an example of what a pool object looks like.